extensions-core/parquet-extensions/pom.xml

```diff
@@ -288,72 +288,70 @@
           <artifactId>jersey-json</artifactId>
         </exclusion>
         <exclusion>
-          <artifactId>log4j</artifactId>
           <groupId>log4j</groupId>
+          <artifactId>log4j</artifactId>
         </exclusion>
         <exclusion>
-          <artifactId>jetty-sslengine</artifactId>
           <groupId>org.mortbay.jetty</groupId>
+          <artifactId>jetty-sslengine</artifactId>
         </exclusion>
         <exclusion>
-          <artifactId>jetty-util</artifactId>
```

Comment:
thanks for fixing the ordering on this, however it looks like a couple of `artifactId` lines got lost

Reply:
Oops, fixed. Thanks!